sphinx/source/docs/user_guide/styling.rst

```diff
@@ -236,23 +236,11 @@ Use one of these three options to define colors in Bokeh:
 
 .. include:: ../includes/colors.txt
 
-.. warning::
```

Comment:
@bryevdv While reviewing some color-related paragraphs of the docs, I came across this warning. I have been able to create a plot with lists of RGB as well as RGBA color tuples as color arguments - am I correct to assume that the issue described here has been fixed sometime in the past?

Reply:
... I believe @jbednar mentioned this as early as 2016 https://github.com/bokeh/bokeh/issues/2622#issuecomment-253008907

Reply:
I do believe *lists* of RGB(a) tuples passed directly or in CDS columns may work now. I am not so sure about passing a *single* tuple, are you saying you checked that as well?

Reply:
I first checked a list of RGB tuples and a list of RGBA tuples. Then, I also passed a single tuple as color directly: `p.circle([1,2,3], [1,2,3], radius=.25, color=(255, 0, 0, 0.5))` (and, just to be sure, also a list containing a single tuple: `p.circle(1, 2, radius=.25, color=[(255, 0, 0, 0.5)])`). All of those worked fine.

Reply:
On interesting observation with HSL values though: while I was able to use a single tuple _and_ a list of tuples without any issues, I was only able to plot a graph with list of HSL values, not with a single HSL value. 

This works:
```
xs = [1,2,3]
ys = [1,2,3]
col = ["hsl(60deg 100% 50% / 1.0)", "hsl(0deg 100% 50% / 1.0)", "hsl(12deg 100% 50% / 1.0)"]
p = figure(title = "HSL lists work")
p.circle(xs, ys, radius=.25, color=col)
```

This raises an error `(BAD_COLUMN_NAME)`:

```
xs = [1,2,3]
ys = [1,2,3]
col = "hsl(60deg 100% 50% / 1.0)"
p = figure(title = "Single HSL value doesn't")
p.circle(xs, ys, radius=.25, color=col)
```

@bryevdv To me, this seems somewhat inconsistent. Is this different behaviour for HSL intentional or something that should be fixed?